op-node/node/node.go

```diff
@@ -635,10 +648,25 @@ func (n *OpNode) Stop(ctx context.Context) error {
 			result = multierror.Append(result, fmt.Errorf("failed to close RPC server: %w", err))
 		}
 	}
+
+	// Stop sequencer and report last hash. l2Driver can be nil if we're cleaning up a failed init.
+	if n.l2Driver != nil && n.cfg.Driver.SequencerEnabled {
+		latestHead, err := n.l2Driver.StopSequencer(ctx)
+		switch {
```

Comment:
You could do this, and then don't need to retain the config in the OpNode struct:
```suggestion
	if n.l2Driver != nil {
		latestHead, err := n.l2Driver.StopSequencer(ctx)
		switch {
		case errors.Is(err, driver.ErrSequencerNotEnabled): // do nothing, also not log
```
`ErrSequencerNotEnabled` is returned by the `DisabledSequencer` on `Stop`.

Reply:
It is preferable it fits the convention here. I've fixed it.